libpod.conf

```diff
@@ -0,0 +1,54 @@
+# libpod.conf is the default configuration file for all tools using libpod to
+# manage containers
+
+# Default transport method for pulling and pushing for images
+image_default_transport = "docker://"
+
+# Paths to look for a valid OCI runtime (runc, runv, etc)
+runtime_path = [
+	     "/usr/bin/runc",
+	     "/usr/sbin/runc",
+	     "/sbin/runc",
+	     "/bin/runc",
+	     "/usr/lib/cri-o-runc/sbin/runc"
+]
+
+# Paths to look for the Conmon container manager binary
+conmon_path = [
+	    "/usr/libexec/crio/conmon",
+	    "/usr/local/libexec/crio/conmon",
+	    "/usr/bin/conmon",
+	    "/usr/sbin/conmon",
+	    "/usr/lib/crio/bin/conmon"
+]
+
+# Environment variables to pass into conmon
+conmon_env_vars = [
+		"PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
+]
+
+# CGroup Manager - valid values are "systemd" and "cgroupfs"
+cgroup_manager = "cgroupfs"
+
+# Directory for persistent libpod files (database, etc)
+static_dir = "/var/lib/containers/storage/libpod"
```

Comment:
Can this be commented out, and we just get it relative to where container storage is configured in /etc/containers/storage.conf, also have a reference to that here.

Reply:
Fixed. We already set it relative to containers/storage in the baked-in defaults, so just commenting this out is enough to get a path relative to containers/storage